client/core/core.go

```diff
@@ -43,13 +42,8 @@ import (
 )
 
 const (
-	keyParamsKey      = "keyParams"
-	conversionFactor  = 1e8
-	regFeeAssetSymbol = "dcr" // Hard-coded to Decred for registration fees, for now.
-
-	// regConfirmationsPaid is used to indicate completed registration to
-	// (*Core).setRegConfirms.
-	regConfirmationsPaid uint32 = math.MaxUint32
+	keyParamsKey     = "keyParams"
+	conversionFactor = 1e8
```

Comment:
Unrelated, but I believe this is a small problem for eth, and possible other coins. We will do calculations in gwei, which is 1e9 wei, and then one eth is 1e9 of that, so a different conversion factor in order to speak in terms of one eth. I think just used for notifications though.

There are probably more instances of this in the webserver js.

Reply:
I was sure we had an issue for this but I cannot find it.  Whenever I see this `conversionFactor = 1e8` I just pretend I didn't see it and run the other way.